src/main/java/ch/admin/bag/covidcertificate/sdk/core/models/certlogic/CertLogicData.kt

```diff
@@ -30,5 +30,7 @@ internal data class CertLogicHeaders(
 	val iat: String?,
 	val exp: String?,
 	val isIsLight: Boolean?, //Must be isIsLight because the Json Serializer strips one "is"
-	val mode: String?
+	val mode: String?,
+	val iss: String? = null,
+	val kid: String? = null,
```

Comment:
Should it be kidBase64?

Reply:
Left as `kid` as iOS does the same: https://github.com/admin-ch/CovidCertificate-SDK-iOS/pull/141/files